test/language/default-parameters/undefined-parameters-doesnt-replace-default.js

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.3.3.7
+description: >
+  Undefined parameters doesn't replace default values.
```

Comment:
The test name/description should concern "undefined arguments", not
    "undefined parameters". And since the subject is plural, the verb should be
    "don't".